src/DataLib/fendl32B_retrofit/gendf_tools.py

```diff
@@ -0,0 +1,201 @@
+# Import packages
+import argparse
+import csv
+import sys
+from logging_config import logger, LoggerWriter
+import ENDFtk
+import contextlib
+import os
+
+# Define an argument parser
+def fendl_args():
```

Comment:
It's most common to put the argument parser in the file that has the main method.  Is there a reason it's here?